docs/user/pings/ping-schedules-and-timings.md

```diff
@@ -0,0 +1,21 @@
+# Ping schedules and timings overview
+
+Full reference details about the [`metrics`](metrics.html) and [`baseline`](baseline.html) ping schedules are detailed elsewhere.
+
+The following diagram shows a typical timeline of a mobile application, when pings are sent and what timing-related information is included.
+
+![ping timeline diagram](ping_timing.svg)
+
+There are two distinct runs of the application, where the OS shutdown the application at the end of Run 1, and the user started it up again at the beginning of Run 2.
+
+There are three distinct foreground sessions, where the application was visible on the screen and the user was able to interact with it.
+
+The rectangles for the `baseline` and `metrics` pings represent the measurement windows of those pings, which always start exactly at the end of the preceding ping.  The `ping_info.start_time` and `ping_info.end_time` metrics included in these pings correspond to these beginning and the end of their measurement windows.
+
+The `baseline.duration` metric (included only in `baseline` pings) corresponds to amount of time the application spent on the foreground, which, since measurement window always extend to the next ping, is not always the same thing as the `baseline` ping's measurement window.
+
+The `submission_timestamp` is the time the ping was received at the telemetry endpoint, added by the ingestion pipeline.  It is not exactly the same as `ping_info.end_time`, since there may be various networking and system latencies both on the client and in the ingestion pipeline (represented by the dotted horizontal line, not to scale).  Also of note is that `start_time`/`end_time` are measured using the client's real-time clock in its local timezone, which is not a fully reliable source of time.
+
+The "Baseline 2" ping illustrates an important corner case. When "Session 2" ended, the OS also shut down the entire process, and the Glean SDK did not have an opportunity to send a `baseline` ping immediately.  In this case, it is sent at the next available opportunity when the application starts up again in "Run 2".  This `baseline` ping is annotated with the reason code `dirty_startup`.
```

Comment:
Thank you for this thoughtful write up and diagram.  I think this document will make the interpretation much clearer.  

For these last two points about 'corner cases' -  do we have any reports that allow us to monitor these effects on an ongoing basis?  It would be nice to link out to some data points on how big of a corner case this is.

Reply:
We don't have anything simple to track this, but it would be good to create that.

It won't be possible to link to them from these docs, though, which are public.